pkg/tagger/collectors/kubelet_extract_test.go

```diff
@@ -325,6 +335,54 @@ func TestParsePods(t *testing.T) {
 				StandardTags: []string{},
 			}},
 		},
+		{
+			desc: "pod with priority class + k8s recommended tags",
+			pod: &kubelet.Pod{
+				Metadata: kubelet.PodMetadata{
+					Name:      "dd-agent-rc-qd876",
+					Namespace: "default",
+					Labels: map[string]string{
```

Comment:
Are these labels relevant to the test? If not, would it make sense to delete them to simplify?

Reply:
Indeed !
That would simplify the test. Done !